styles/pages/portfolio.scss

```diff
@@ -1,141 +1,250 @@
 .portfolio {
-   padding: 60px 30px;
-   position: relative;
-   @media (max-width: 500px) {
-      padding: 40px 20px;
-   }
-}
-.portfolio__background {
-   min-height: 100vh;
-   position: absolute;
-   left: 0;
-   right: 0;
-   top: 0;
-   bottom: 0;
-   z-index: -1;
-   background: linear-gradient(135deg, #071e40 0%, #662938 80%);
+	padding: 0 30px;
+	position: relative;
+	@media (max-width: 500px) {
+		padding: 40px 20px;
+	}
 }
+
 .portfolio__header {
-   margin-bottom: 50px;
+	@media (max-width: 574.98px) {
```

Comment:
Why do you use so different points? Please talk with UI/UX designer and make variables for these points)

Reply:
Those was copypasted from Anton's branch for redesigning blog. I'll use breakpoint mixin instead.